core/frontend/src/tile/map/ImageryTileTree.ts

```diff
@@ -44,11 +45,20 @@ export class ImageryMapTile extends RealityTile {
     this.setIsReady();
   }
 
-  public selectCartoDrapeTiles(drapeTiles: ImageryMapTile[], rectangleToDrape: MapCartoRectangle, drapePixelSize: number, args: TileDrawArgs): TileTreeLoadStatus {
+  public selectCartoDrapeTiles(drapeTiles: ImageryMapTile[], rectangleToDrape: MapCartoRectangle, drapePixelSize: number, args: TileDrawArgs, context: TraversalSelectionContext): TileTreeLoadStatus {
     // Base draping overlap on width rather than height so that tiling schemes with multiple root nodes overlay correctly.
     if (this.isLeaf || (this.rectangle.xLength() / this.maximumSize) < drapePixelSize || this._anyChildNotFound) {
-      if (this.isDisplayable && !this.isNotFound)
-        drapeTiles.push(this);
+      if (this.isDisplayable
+        && !this.isNotFound
+      ) {
+        if (this.isOutOfLodRange) {
+          context.outOfLodRange.push(this);

```

Comment:
You don't seem to ever use this. I assume you intend to use it in a subsequent PR for UI feedback, but it's unclear how you're going to get it from TraversalSelectionContext to the UI. Remove until subsequent PR?